yarn.lock

```diff
@@ -14504,6 +14514,15 @@ terser@5.3.8, terser@^5.3.8:
     source-map "~0.7.2"
     source-map-support "~0.5.19"
 
+terser@5.4.0:
```

Comment:
This makes me super sad... the fact that `gulp-terser` requires `terser@5.4.0` and yet we apparently use `4.8.0` befuddles me 😂

Reply:
`html-webpack-plugin` (last published 10 days ago 🎉) and more directly `html-minifier-terser` (last published 7 months ago 😢) are the main culprits:

```
yarn why terser                                                                                                                                             0.29s [wincent/fix-ci] ~/code/liferay-frontend-projects
yarn why v1.22.5
[1/4] 🤔  Why do we have the module "terser"...?
[2/4] 🚚  Initialising dependency graph...
warning Missing version in workspace at "/Users/greghurrell/code/liferay-frontend-projects/projects/js-themes-toolkit", ignoring.
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "terser@5.3.8"
info Has been hoisted to "terser"
info Reasons this module exists
   - "workspace-aggregator-2b18f523-66b8-4ae1-9a79-f299cab4ade8" depends on it
   - Hoisted from "_project_#@liferay#npm-scripts#terser"
   - Hoisted from "_project_#terser-webpack-plugin#terser"
info Disk size without dependencies: "2.95MB"
info Disk size with unique dependencies: "3.94MB"
info Disk size with transitive dependencies: "3.95MB"
info Number of shared dependencies: 4
=> Found "gulp-terser#terser@5.4.0"
info This module exists because "_project_#liferay-theme-tasks#gulp-terser" depends on it.
info Disk size without dependencies: "2.61MB"
info Disk size with unique dependencies: "3.6MB"
info Disk size with transitive dependencies: "3.61MB"
info Number of shared dependencies: 4
=> Found "html-minifier-terser#terser@4.8.0"
info This module exists because "_project_#@liferay#npm-scripts#html-webpack-plugin#html-minifier-terser" depends on it.
info Disk size without dependencies: "1.86MB"
info Disk size with unique dependencies: "2.85MB"
info Disk size with transitive dependencies: "2.87MB"
info Number of shared dependencies: 4
✨  Done in 0.96s.
```

And people wonder why I am so grumpy about dependencies...